core/shared/src/main/scala/scodec/bits/ByteVector.scala

```diff
@@ -1218,6 +1233,13 @@ object ByteVector {
         i += 1
       }
     }
+    def copyToBuffer(buffer: ByteBuffer, offset: Long, size: Int): Unit = {
+      var i = 0
+      while (i < size && buffer.remaining > 0) {
```

Comment:
Can we avoid the while loop and simply call the `ByteBuffer::put(Array[Byte], offset, len)` overload?